CMakeLists.txt

```diff
@@ -23,12 +23,15 @@ include(CTest)
 include(test)
 
 # Options
-OPTION(WLC_X11_SUPPORT "Build X11 backend and XWayland support" ON)
+OPTION(WLC_X11_BACKEND_SUPPORT "Build X11 backend" OFF)
+OPTION(WLC_XWAYLAND_SUPPORT "Support XWayland applications" OFF)
```

Comment:
Aaah, I thought OFF was the default (probably edited it myself while testing).
Thanks!